### ISSUE TITLE
Project Testing

### ISSUE THREAD

**Issue by miguelmoraleda** [Jam3/jam3-testing-tools#1]
Would be nice to have more info about testing the projects on different browsers/devices. How to use debug on devices, etc.

**Comment by mattdesl** [comment]
Yeah! :+1:  This could be a pretty big topic when you get into remote debugging for android etc. We could have it in a different repo or a different markdown file.
